### ISSUE TITLE
[bug] No Projects on set up causes 500 error

### ISSUE THREAD

**Issue by antdke** [ryanckulp/10k_hours#19]
hey ryan, 

ran into this error when i cloned the repo and when through the set up instructions in `ReadMe.md`.

seems like since there are 0 Projects when you start running the code, it'll crash. not sure if that's a big issue for you. 

one solution could be to add `rails db:seeds` in `bin/setup`? like in foundr digest

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/22419667/212435625-d4fe0dea-f24f-4762-afb6-8781f47168ac.png">

**Comment by ryanckulp** [comment]
hey @antdke , did you follow the readme instructions? 

Step 5 (`rails db:setup`) _should_ invoke `db:seed`, which will use the values inside `default_values.rb` to create initial records in the database ([more info](https://github.com/ryanckulp/10k_hours/blob/2208beade1e348a0d892f5dda1a8dfd7ed6041f3/bin/setup#L26)).

**Comment by antdke** [comment]
@ryanckulp yup! followed the readme instructions. even triple-checked (cloned the repo and set it up a few times)

but, good news: i found a bug in `db/seeds.rb` that was causing this.

i changed `Project.current` to `Project.first` and that seemed to resolve the issue. 

✅  a user can now run `rails db:setup` and a Project will be created on setup

(my pull request for this change --> #20 )
